README.md

```diff
@@ -16,7 +16,7 @@ a direct visit from the browser.
 
 ```javascript
 // Heroku defines the environment variable PORT, and requires the binding address to be 0.0.0.0
-var host = process.env.PORT ? '0.0.0.0' : '127.0.0.1';
+var host = process.env.HOST || '127.0.0.1';
```

Comment:
Where is this "HOST" environment variable coming from?

This example is specific to Heroku (and a simplified version of the server.js file), but it can easily be tweaked. If the change is really necessary, the fallback should be '0.0.0.0', not '127.0.0.1', and both README.md and server.js should be edited.